### ISSUE TITLE
Error occurred when executing AnimateDiffSampler:  BaseModel has no attribute `betas`

### ISSUE THREAD

**Issue by wangminrui2022** [ArtVentureX/comfyui-animatediff#61]
Error occurred when executing AnimateDiffSampler:

BaseModel has no attribute `betas`

File "E:\cmdmax-web\program\2023-v1\ComfyUI-App-Win\ComfyUI\execution.py", line 153, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "E:\cmdmax-web\program\2023-v1\ComfyUI-App-Win\ComfyUI\execution.py", line 83, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "E:\cmdmax-web\program\2023-v1\ComfyUI-App-Win\ComfyUI\execution.py", line 76, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "E:\cmdmax-web\program\2023-v1\ComfyUI-App-Win\ComfyUI\custom_nodes\comfyui-animatediff\animatediff\sampler.py", line 288, in animatediff_sample
model = self.inject_motion_module(model, motion_module, inject_method, video_length)
File "E:\cmdmax-web\program\2023-v1\ComfyUI-App-Win\ComfyUI\custom_nodes\comfyui-animatediff\animatediff\sampler.py", line 218, in inject_motion_module
self.override_beta_schedule(model.model)
File "E:\cmdmax-web\program\2023-v1\ComfyUI-App-Win\ComfyUI\custom_nodes\comfyui-animatediff\animatediff\sampler.py", line 189, in override_beta_schedule
self.prev_beta = model.get_buffer("betas").cpu().clone().detach()
File "E:\cmdmax-web\program\2023-v1\ComfyUI-App-Win\ComfyUI\venv\lib\site-packages\torch\nn\modules\module.py", line 750, in get_buffer
raise AttributeError(mod._get_name() + " has no attribute `"

**Comment by wangminrui2022** [comment]
如何解决这个问题呢？
How to solve this problem?
この問題を解決するにはどうすればよいでしょうか?

**Comment by shir0tetsuo** [comment]
I am having similar issue:

Error occurred when executing AnimateDiffSampler:

BaseModel has no attribute `betas`

  File "/home/inpw/ComfyUI/execution.py", line 153, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "/home/inpw/ComfyUI/execution.py", line 83, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "/home/inpw/ComfyUI/execution.py", line 76, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
  File "/home/inpw/ComfyUI/custom_nodes/comfyui-animatediff/animatediff/sampler.py", line 288, in animatediff_sample
    model = self.inject_motion_module(model, motion_module, inject_method, video_length)
  File "/home/inpw/ComfyUI/custom_nodes/comfyui-animatediff/animatediff/sampler.py", line 218, in inject_motion_module
    self.override_beta_schedule(model.model)
  File "/home/inpw/ComfyUI/custom_nodes/comfyui-animatediff/animatediff/sampler.py", line 189, in override_beta_schedule
    self.prev_beta = model.get_buffer("betas").cpu().clone().detach()
  File "/home/inpw/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 750, in get_buffer
    raise AttributeError(mod._get_name() + " has no attribute `"

**Comment by Theodotus1243** [comment]
The same issue

**Comment by artventuredev** [comment]
@wangminrui2022 @shir0tetsuo @Theodotus1243 This issue should be fixed on the latest update.

**Comment by invincibledinku** [comment]
![image](https://github.com/ArtVentureX/comfyui-animatediff/assets/6459572/d4dfdec0-13c2-4be3-9173-e5355b9e8a9d)

**Comment by artventuredev** [comment]
@invincibledinku Based on your logs I can tell that you're not on the latest version, please do the update and try again.
